content/Store Support/Shipping/If-you-havent-received-your-delivery.md

```diff
@@ -0,0 +1,24 @@
+---
+title: If you haven't received your delivery
+id: 14085643116700
+---
+
+In rare cases, an order might not reach its destination even when the tracking displays it as delivered. This is more common with shipping methods that don't offer proof of delivery.
+
+Here are a few steps to follow if a delivery is missing:
+
```

Comment:
I think we should include a "Check your item's delivery information" as the first step, and not assume that the customer has already done this. We can link to https://support.arduino.cc/hc/en-us/articles/13756849779228-Track-your-order